pages/pinner-award.md

```diff
@@ -0,0 +1,51 @@
+title: The John Pinner Award
+---
+<h2>What?</h2>
+
+The annual John Pinner award celebrates voluntary service to the Python
+community in the UK. It's for those who freely contribute time and effort for
+the good of the community but who generally don't get the sort of recognition
+others may enjoy.
+
+This is an annual award with up to three awards being given at once. The awards
+will be announced at the start of each annual PyCon UK conference, so
+recipients at the conference may enjoy a weekend of recognition from the wider
+community in attendence.
+
+It is only possible to win the award once.
+
+Recipients of the award will receive a free ticket to next year's PyCon UK and
+a token in recognition of their efforts (most likely a medal of some sort).
+
+<h2>Why?</h2>
+
+John Pinner founded and organised PyCon UK until he passed away in 2015. He was
+the personification of a Pythonista who gave time, code and effort back to the
+community. He was a role model, mentor and teacher who is sorely missed by
+many.
+
+With the blessing of the Pinner family, the UKPA instigating an award to
```

Comment:
nit: "the UKPA _is_ instigating an award"?

(also not a fan of "instigating", although struggling to think of a good alternative)